lib/Dialect/Comb/CombFolds.cpp

```diff
@@ -827,6 +849,14 @@ OpFoldResult AndOp::fold(FoldAdaptor adaptor) {
     }
   }
 
+  // x0 = icmp(x, y)
+  // x1 = n_icmp(x, y)
```

Comment:
nit: `n_icmp` might not be easy to understand at the first look so could you write more descriptive comment here like `icmp(pred, x, y), icmp(nagated(pred), x, y)) -> 0`?

Reply:
I've changed it to `pred` and `!pred` which hopefully should be clear enough.